src/tools/validators.js

```
@@ -0,0 +1,129 @@
+/*
+    Champs formations sans validation connue
+    "accessor": "nom_academie",
+    "accessor": "ds_id_dossier",    
+    "accessor": "etablissement_responsable_siret_intitule",    
+    "accessor": "etablissement_formateur_siret_intitule",
+    "accessor": "etablissement_reference_type",
+    "accessor": "etablissement_reference_conventionne",
+    "accessor": "etablissement_reference_declare_prefecture",
+    "accessor": "etablissement_reference_datadock",
+    "accessor": "source",
+    "accessor": "diplome",
+    "accessor": "intitule",
+    "accessor": "nom_academie_siege",
+
+  Champs établissements sans validation connue
+
+    "accessor": "computed_type",
+    "accessor": "computed_conventionne",
+    "accessor": "computed_declare_prefecture",
+    "accessor": "computed_info_datadock",
+    "accessor": "capital_social",
+    "accessor": "numero_tva_intracommunautaire",
+    "accessor": "forme_juridique",
+    "accessor": "forme_juridique_code",
+    "accessor": "ds_questions_nom",
+    "accessor": "raison_sociale",
+    "accessor": "code_effectif_entreprise",
+    "accessor": "date_creation",
+    "accessor": "nom",
+    "accessor": "prenom",
+    "accessor": "naf_code",
+    "accessor": "naf_libelle",
+    "accessor": "adresse",
+    "accessor": "numero_voie",
+    "accessor": "type_voie",
+    "accessor": "nom_voie",
+    "accessor": "complement_adresse",
+    "accessor": "localite"
+*/
+
+
+
+export const validateCell = (accessor, value) => {
+  if (fieldValidators[accessor]) return fieldValidators[accessor](value);
+  else return true;
+};
```

Need to throw "Filed unknown" and return false pour qu'au niveau dev on puisse voir que qqch ne va pas

---

Tous les champs n'ont pas de validators. Du coup je verrai la choses suivante : 
- si pas de validator -> le test passe
- si un validateur et mauvais format -> le test ne passe pas
- si un validateur et bon format -> le test passe
- si un validateur et throw exception -> le test ne passe pas . Et pour qu'on le voit juste un console log en attendant d'avoir Sentry ?

---

Je pense que ça serait mieux de créer des validator pour tous les champs. Et puisque  qu'on a pas encore réfléchi à la règle de validation de ces champs les validators return true en permanence. Et on laisse ta liste des validators à faire (très utile selon moi).
et du coup pas de règles spécifique sur l'algo:

- si un validateur et mauvais format -> le test ne passe pas 
- si un validateur et bon format -> le test passe 
- si pas de validateur -> throw new Error +  console.log (le temps de mettre sentry) = le test ne passe pas .

---

OK vu. je fais ça.